openreview/journal/journal.py

```diff
@@ -449,7 +452,12 @@ def get_late_invitees(self, invitation_id):
                     signature_members.append(signature)
                 else:
                     signature_members = signature_members + signature['members']
+            for signature in reply.signatures:
```

Comment:
@carlosmondra this is the fix I'm doing.

Reply:
This code is to check who are the late invitees. in the case where the signature is a tilde id, I don't need to check the signature in details, I can check the reply.signatures instead.